src/lib.rs

```diff
@@ -1536,6 +1599,22 @@ mod test {
         assert_eq!(format!("{}", Ratio::from_integer(-2)), "-2".to_string());
     }
 
+    #[test]
+    #[cfg(feature = "std")]
+    fn test_as_decimal() {
+        use AsDecimal;
+        use std::string::ToString;
+        assert_eq!(_2.as_decimal(2), "2".to_string());
```

Comment:
Shouldn't we output "2.00" for this?

Reply:
For some reason I assumed that it's expected to return value without zeroes if there's nothing beside them after the separator. Fixed.